guides/pro/authorization.md

```diff
@@ -93,8 +93,7 @@ end
 Also, you can specify authentication at __type-level__, for example:
 
 ```ruby
-AccountBalanceType = GraphQL::ObjectType.define do
-  name "AccountBalance"
+class AccountBalanceType < GraphQL::ObjectType
```

Comment:
These should all be `GraphQL::Schema::Object` throughout

Reply:
Just fixed them!